libstorage/api/context/context.go

```diff
@@ -1,14 +1,14 @@
 package context
 
 import (
+	"context"
```

Comment:
so is this possible because we are only on GO 1.[7-9] now? If so, do we need to update `travis.yml` to reflect that?

Reply:
Hi @codenrhoden,

The `.travis.yml` Go version is 1.8.3 as we discussed.